askbot/jinja2/widgets/secondary_header.html

```diff
@@ -38,18 +38,11 @@
                             <i class="rover-icon rover-icon-house"></i>
                         </a>
                     </td>
-                    <td width="1">{% include "widgets/scope_nav.html" %}</td>
-                    {# width * means that the cell takes the remaining table width #}
-                    <td width="*" class="search-bar">{% include "widgets/search_bar.html" %}</td>
-                    {% if settings.ASK_BUTTON_ENABLED %}
```

Comment:
I looked for this setting, but couldn't find where to change it. Decided it was equivalent to just remove the whole block.

Reply:
Askbot does weird things with settings. For example, it provides a way to set the from the Django admin interface (which is a terrible idea and against Django principles -- it probably wouldn't work in newer Django versions either way because of how the code is loaded).